public/style.css

```diff
@@ -289,6 +289,8 @@ textarea::placeholder {
   position: relative;
   z-index: 10;
   flex-shrink: 0;
+  transition: background 0.15s;
+  margin: 0 -2px;
 }
 
 .resize-handle::after {
```

Comment:
the resize handle already existed as a css class but wasnt being used, it appears